### PR TITLE
Removing obsolete & duplicate resources

### PR DESCRIPTION
**Pull request by dimitar-kostadinov**
**What this PR does / why we need it**:

Removing obsolete & duplicate resources to avoid collisions, e.g. under https://gardener.cloud/docs/guides/administer_shoots/ there are two "Trigger Shoot operations" guides.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

File: .docforge/guides.yaml [hunk 1]
```diff
@@ -1,5 +1,3 @@
-nodesSelector:
```
Comment (Kristian-ZH) [inline .docforge/guides.yaml:1]: why don't we use the nodesSelector instead of defining a custom structure which will look in the same way at the end?

Comment (dimitar-kostadinov) [reply .docforge/guides.yaml:1]: If the nodeSlector is on the top-level path, the following node collisions are visible:
```
In docs.guides container node.
  Node with name install_gardener appears 2 times for sources: , . 
  Node with name administer_shoots appears 2 times for sources: , . 
  Node with name applications appears 2 times for sources: , .
```
To resolve these conflicts, a separate node selector is defined for each sublevel. 
Once the node merging functionality is ready, such node collisions will not occur and the nodeSelector can be returned back to the top-level.

Comment (JordanJordanov) [reply .docforge/guides.yaml:1]: I propose to remove the older (obsolete) versions of the duplicated pages

Comment (Kristian-ZH) [reply .docforge/guides.yaml:1]: Yes I also think that we should remove them but also in my opinion, the newer versions of the pages should be moved to https://github.com/gardener/documentation/tree/master/website/documentation/guides

In this way we will have just one nodeSelector which will selects all of the up-to-date pages.

**Review by JordanJordanov — APPROVED** [review]

**Review by Kristian-ZH — APPROVED** [review]
/lgtm

**Pull request merged**